Gemfile

```diff
@@ -9,3 +9,10 @@ unless RUBY_PLATFORM
   gem 'pry-stack_explorer'
   gem 'pry-rescue'
 end
+
+# Gem a dependency of Capybara, new version of rack 2.0 onwards require ruby 2.2 and above.
+if RUBY_VERSION < "2.2.2"
```

Comment:
This does  a string comparison, so '2.20.0' might trigger (not sure). But unlikely to be an issue for a long time.